lib/models/Book.js

```diff
@@ -18,15 +19,15 @@ module.exports = class Book {
 
   static async getById(id) {
     const { rows } = await pool.query(
-      `SELECT books.*,
+      `SELECT books.title, books.released,
         COALESCE(
-          json_agg(to_jsonb(authors))
+          json_agg(json_build_object('id', authors.id, 'name', authors.name))
           FILTER (WHERE authors.id IS NOT NULL), '[]'
         ) as authors from books
         LEFT JOIN books_authors
           ON books.id = books_authors.book_id
         LEFT JOIN authors
-          ON books_authors.authors_id = authors.id
+          ON books_authors.author_id = authors.id
           WHERE books.id = $1
           GROUP BY books.id`,
```

Comment:
There should be no need to 'GROUP' here because you are only expecting one row to be returned